halo2_proofs/src/dev/failure/emitter.rs

```diff
@@ -11,6 +11,7 @@ use crate::{
 
 fn padded(p: char, width: usize, text: &str) -> String {
     let pad = width - text.len();
+
```

Comment:
I have not tested against the code in this PR so this could already be fixed, but on the PSE fork `padded()` is now also used in cases where `text.len() > width`, which will make this subtraction panic. The easiest example of this is when the column name is only a single character, which makes `width = 1`, and then something like `text = "x1"` needs to be padded and written in this column.

Even if fixed in some other way, some kind of `if` statement mitigating this problem that may not produce the nicest output but at least never panics no matter the inputs would be a good idea I think.